[HW] 1072938

Yuming018 opened the pull request:
Signed-off-by: 1072938 <s1072938@mail.ncyu.edu.tw>

jrjang commented:
Hi @1072938 ,

commit message 格式有誤
1. commit summary 為 [HW] 學號
2. commit message 為解決問題與方法

Yuming018 commented:
修改完格式了，還請老師檢查一下，麻煩老師了

jrjang commented:
Hi @1072938 ,

commit message 描述與實際修改不符. 麻煩確認一下

Yuming018 commented:
修改完了，還請老師檢查一下，老師辛苦了

jrjang commented on README.md:
````diff
@@ -9,7 +9,13 @@ cd ncyu-2021-ex1
 echo "Example 1" > ex1.txt
 ```
-
+可以改成以下指令
````
1. 上面空一行
2. 此行敘述改成 "或者"

jrjang commented on README.md:
````diff
@@ -9,7 +9,13 @@ cd ncyu-2021-ex1
-
+可以改成以下指令
+```
+git clone -b ex1 https://github.com/1072938/ncyu-2021.git ncyu-2021-ex1
````
1072938 改成 GITHUB_ACCOUNT

jrjang commented on README.md:
````diff
@@ -9,7 +9,13 @@ cd ncyu-2021-ex1
+```
+git clone -b ex1 https://github.com/1072938/ncyu-2021.git ncyu-2021-ex1
+cd ncyu-2021-ex1
+git init
````
不用再做 `git init`

jrjang commented on README.md:
````diff
@@ -9,7 +9,13 @@ cd ncyu-2021-ex1
+cd ncyu-2021-ex1
+git init
+echo "Example 1" > ex1.txt
+```
````
以下要空一行, 再加上 GITHUB_ACCOUNT 說明，再空一行

jrjang commented:
Hi @1072938 ,

commit message 請再描述清楚

Yuming018 commented:
再麻煩老師幫我檢查一下，謝謝

jrjang commented on README.md:
````diff
@@ -10,6 +10,15 @@ git init			# 執行 git init
+echo "Example 1" > ex1.txt
+```
+
+GITHUB_ACCOUNT: github帳號
````
中英文之間需要一個空白

jrjang commented:
Hi @1072938 ,

commit message 可以改成:

除了使用 git init 建立 git project 外，在 README.md 中，加入 git clone
指令來下載 github 上的 ncyu-2021 project

Yuming018 commented:
謝謝老師檢查這麼多次

Commit:
[HW] 1072938

除了使用 git init 建立 git project 外，在 README.md 中，加入 git clone
指令來下載 github 上的 ncyu-2021 project

Signed-off-by: 1072938 <s1072938@mail.ncyu.edu.tw>

jrjang commented:
Good job.
